spec/factories/questions.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :question do
+    problem_id 1
+    name "MyString"
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.